README.md

```diff
@@ -293,25 +286,22 @@ Make sure Fabric is in your ``$GOPATH`` and you enable the plugin feature using
 
 Building Fabric may take a while and it's time to get a coffee. Also, be not surprised if unit tests fail. In order to
 just build the peer you can run the following command:
-
-    $ GO_TAGS=pluginsenabled make peer
-
+```
+$ GO_TAGS=pluginsenabled make peer
+```
 Please make sure that the peer is _always_ built with GO_TAGS, otherwise our custom validation plugins will (silently!)
 ignored by the peer, despite the settings in ``core.yaml``.
 
+#### Build the entire project
```

Comment:
this is also not on the level of option 2 but does apply to both options, so should be one level up? Or maybe add new third-level section like Developping in FPC or alike with this and below as sub-sections (on same level as before). In that case you could potentially also move the IAS section here as first section or so

The "not restricted to option 2" of course also applies to the following section up to the "Developping your first private chaincode"

Reply:
If GO_TAGS is required, can't it be automatically set in the docker development environment?  All the other required ENV variables are set.

Reply:
I have to admit i don't know the exact semantics of GO_TAGS. Is this a Fabric specific variable or something generic go? In either case, it could potentially negatively affect our own FPC go builds?  So i would be a bit reluctant to that to the docker image environment